modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/carbontools/ChangeUserPasswordMySQLDBTestCase.java

```diff
@@ -0,0 +1,130 @@
+
+import org.apache.axis2.AxisFault;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.appserver.integration.common.utils.ASIntegrationConstants;
+import org.wso2.appserver.integration.common.utils.ASIntegrationTest;
+import org.wso2.appserver.integration.common.utils.CarbonCommandToolsUtil;
+import org.wso2.carbon.automation.engine.annotations.ExecutionEnvironment;
+import org.wso2.carbon.automation.engine.annotations.SetEnvironment;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.ContextXpathConstants;
+import org.wso2.carbon.automation.engine.frameworkutils.enums.OperatingSystems;
+import org.wso2.carbon.integration.common.admin.client.AuthenticatorClient;
+import org.wso2.carbon.integration.common.extensions.usermgt.UserPopulator;
+import org.wso2.carbon.integration.common.tests.CarbonTestServerManager;
+
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.util.HashMap;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * This class is to test change MySQL user password using chpasswd.sh/chpasswd.bat
+ * All the test cases in this class are disabled because need to test this with mysql
+ */
+public class ChangeUserPasswordMySQLDBTestCase extends ASIntegrationTest {
+
+    private static final Log log = LogFactory.getLog(ChangeUserPasswordMySQLDBTestCase.class);
+    private boolean scriptRunStatus;
+    private AutomationContext context;
+    private int portOffset = 1;
+    private HashMap<String, String> serverPropertyMap = new HashMap<String, String>();
+    private String carbonHome = null;
+    private static String MYSQL_DB_URL;
+    private AuthenticatorClient authenticatorClient;
+    private static final char[] userNewPassword = {'t', 'e', 's', 't', 'u', '1', '2', '3'};
+    private static final String userName = "testu1";
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws XPathExpressionException, AxisFault {
+        String PRODUCT_NAME = "AS";
+        String INSTANCE = "appServerInstance0002";
+        context = new AutomationContext(PRODUCT_NAME, INSTANCE,
+                                        ContextXpathConstants.SUPER_TENANT,
+                                        ContextXpathConstants.SUPER_ADMIN);
+        authenticatorClient = new AuthenticatorClient(context.getContextUrls().getBackEndUrl());
+        MYSQL_DB_URL = context.getConfigurationValue(String.format(ASIntegrationConstants.CONTEXT_XPATH_DB_CONNECTION_URL, "MySQL"));
+    }
+
+    @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE})
+    @Test(groups = "wso2.as", description = "H2DB Password changing script run test", enabled = false)
+    public void testScriptRunChangeUserPasswordH2DB() throws Exception {
```

Comment:
Method name indicate this is H2DB sepecific test, however you use MYSQL_DB_URL in test method body.